packages/mobile/src/account/Settings.tsx

```diff
@@ -0,0 +1,344 @@
+      })
+    }
+  }
+
+  goToProfile = () => {
+    CeloAnalytics.track(CustomEventNames.edit_profile)
+    this.props.navigation.navigate(Screens.Profile)
+  }
+
+  goToLanguageSetting = () => {
+    this.props.navigation.navigate(Screens.Language, { fromSettings: true })
+  }
+
+  goToLocalCurrencySetting = () => {
+    this.props.navigation.navigate(Screens.SelectLocalCurrency)
+  }
+
+  goToLicenses = () => {
+    this.props.navigation.navigate(Screens.Licenses)
+  }
+
+  goToSupport = () => {
+    this.props.navigation.navigate(Screens.Support)
+  }
+
+  resetAppOpenedState = () => {
+    this.props.resetAppOpenedState()
+    Logger.showMessage('App onboarding state reset.')
+  }
+
+  toggleNumberVerified = () => {
+    this.props.setNumberVerified(!this.props.numberVerified)
+  }
+
+  revokeNumberVerification = async () => {
+    if (this.props.e164PhoneNumber && !isE164Number(this.props.e164PhoneNumber)) {
+      Logger.showMessage('Cannot revoke verificaton: number invalid')
+      return
+    }
+    Logger.showMessage(`Revoking verification`)
+    this.props.revokeVerification()
+  }
+
+  toggleBackupState = () => {
+    this.props.toggleBackupState()
+  }
+
+  showDebugScreen = () => {
+    this.props.navigation.navigate(Screens.Debug)
+  }
+
+  getDevSettingsComp() {
+    const { devModeActive } = this.props
+
+    if (!devModeActive) {
+      return null
+    } else {
+      return (
+        <View style={styles.devSettings}>
+          {/* <View style={style.devSettingsItem}>
+            <TouchableOpacity onPress={this.revokeNumberVerification}>
+              <Text>Revoke Number Verification</Text>
+            </TouchableOpacity>
+          </View> */}
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={this.toggleNumberVerified}>
+              <Text>Toggle verification done</Text>
+            </TouchableOpacity>
+          </View>
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={this.resetAppOpenedState}>
+              <Text>Reset app opened state</Text>
+            </TouchableOpacity>
+          </View>
+
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={this.toggleBackupState}>
+              <Text>Toggle backup state</Text>
+            </TouchableOpacity>
+          </View>
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={this.showDebugScreen}>
+              <Text>Show Debug Screen</Text>
+            </TouchableOpacity>
+          </View>
+          <View style={styles.devSettingsItem}>
+            <TouchableOpacity onPress={Sentry.nativeCrash}>
+              <Text>Trigger a crash</Text>
+            </TouchableOpacity>
+          </View>
+        </View>
+      )
+    }
+  }
+
+  disableFornoMode = () => {
+    this.props.toggleFornoMode(false)
+    this.hideFornoSwitchOffWarning()
+  }
+
+  handleFornoToggle = (fornoMode: boolean) => {
```

Comment:
@annakaz do you know if this is still required to prevent crashes? thought we might be okay now

Reply:
I think @etuleu was looking into this issue https://github.com/celo-org/celo-monorepo/issues/2101 , but I have not checked whether it reproduces recently